packages/xo-web/src/xo-app/host/tab-patches.js

```diff
@@ -60,6 +60,77 @@ const MISSING_PATCH_COLUMNS = [
   },
 ]
 
+const MISSING_PATCH_COLUMNS_XCP = [
+  {
+    name: _('patchNameLabel'),
+    itemRenderer: patch => patch.name,
+    sortCriteria: 'name',
+  },
+  {
+    name: _('patchDescription'),
+    itemRenderer: patch => patch.description,
+    sortCriteria: 'description',
+  },
+  {
+    name: _('patchVersion'),
+    itemRenderer: patch => patch.version,
+  },
```

Comment:
Not sortable?
Is it by design?

Reply:
I don't think you can provide a custom comparison function to `_.orderBy` (which is used for sorting) so I'm not sure if it's possible.

Reply:
https://github.com/omichelsen/compare-versions

Reply:
Or even `semver/compare`, which is already a dependency.

Reply:
Yes I get that, but I don't think there's a way to *pass* a compare function to the `sortCriteria`. You can only pass a property name or a getter.

Reply:
https://github.com/vatesfr/xen-orchestra/blob/f5bff408a8656fb8ae13824c100924a178259070/packages/xo-web/src/common/sorted-table/index.js#L294 :slightly_smiling_face:

Reply:
Hmm, you may be right :stuck_out_tongue:

Reply:
Let's forget this for now.